tests/test_benchmark.py

```diff
@@ -341,7 +341,7 @@ def build(self, parent, filename, args, shared_args, emcc_args, native_args, nat
         try_delete(dir_)
 
   def run(self, args):
-    return jsrun.run_js(self.filename, engine=self.engine, args=args, stderr=PIPE, full_output=True, assert_returncode=None)
+    return jsrun.run_js(self.filename, engine=self.engine, args=args, stderr=PIPE, full_output=True, assert_returncode=jsrun.NON_ZERO)
```

Comment:
we don't test the benchmarks on CI, are you sure this is right? I'm surprised this expects a failure.

Reply:
I'm changed to expect success.   All the other benchmark runners seems to expect success so I don't see why chreep wouldn't.   I'm basically removing the ability to be ambivalent about this, so hopefully its easy enough to make sure the cheerp returns zero from these tests.